.gitignore

```diff
@@ -11,5 +11,6 @@ toolchain/*/work*
 toolchain/*/*_done
 toolkit/*/work*
 local.mk
+.idea/
 *.DS_Store
```

Comment:
Why `.idea` ?

Reply:
It's the PyCharm project directory, will remove that,